modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/action/executor/test/ObjectActionExecutorRegistryImplTest.java

```diff
@@ -0,0 +1,161 @@
+	public static void setUpClass() throws Exception {
+		_objectDefinition = ObjectDefinitionTestUtil.addObjectDefinition(
+			_objectDefinitionLocalService);
+
+		Bundle bundle = FrameworkUtil.getBundle(
+			ObjectActionExecutorRegistryImplTest.class);
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
+		_serviceRegistration = bundleContext.registerService(
+			ObjectActionExecutor.class,
+			new TestObjectActionExecutorImpl(
+				SetUtil.fromArray(_objectDefinition.getName()),
+				_OBJECT_ACTION_EXECUTOR_KEY),
+			new HashMapDictionary<>());
+	}
+
+	@AfterClass
+	public static void tearDownClass() throws Exception {
+		_serviceRegistration.unregister();
+
+		_objectDefinitionLocalService.deleteObjectDefinition(
+			_objectDefinition.getObjectDefinitionId());
+	}
+
+	@Test
+	public void testGetObjectActionExecutor() {
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_ADD_OBJECT_ENTRY));
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_GROOVY));
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_NOTIFICATION));
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_UPDATE_OBJECT_ENTRY));
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_WEBHOOK));
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				_OBJECT_ACTION_EXECUTOR_KEY));
+	}
+
+	@Test
+	public void testGetObjectActionExecutors() throws PortalException {
+		ObjectDefinition userObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), User.class.getName());
+
+		List<ObjectActionExecutor> objectActionExecutors =
+			_objectActionExecutorRegistry.getObjectActionExecutors(
+				userObjectDefinition.getName());
+
+		Assert.assertEquals(
+			objectActionExecutors.toString(), 5, objectActionExecutors.size());
+
+		List<String> objectActionExecutorsNames = ListUtil.toList(
+			objectActionExecutors, ObjectActionExecutor::getKey);
+
+		Assert.assertFalse(
+			objectActionExecutorsNames.contains(_OBJECT_ACTION_EXECUTOR_KEY));
+
+		objectActionExecutors =
+			_objectActionExecutorRegistry.getObjectActionExecutors(
+				_objectDefinition.getName());
+
+		Assert.assertEquals(
+			objectActionExecutors.toString(), 6, objectActionExecutors.size());
+
+		objectActionExecutorsNames = ListUtil.toList(
+			objectActionExecutors, ObjectActionExecutor::getKey);
+
+		Assert.assertTrue(
+			objectActionExecutorsNames.contains(_OBJECT_ACTION_EXECUTOR_KEY));
+	}
+
+	private static final String _OBJECT_ACTION_EXECUTOR_KEY =
+		RandomTestUtil.randomString();
+
+	@Inject
+	private static ObjectActionLocalService _objectActionLocalService;
```

Comment:
@murilostodolni do we need this here?

Reply:
It is no longer necessary. I will remove.